libmpv/ios/ViewController.m

```diff
@@ -0,0 +1,209 @@
+@import GLKit;
+@import OpenGLES;
+
+#import "ViewController.h"
+
+
+#import <mpv/client.h>
+#import <mpv/opengl_cb.h>
+
+#import <stdio.h>
+#import <stdlib.h>
+
+
+static inline void check_error(int status)
+{
+    if (status < 0) {
+        printf("mpv API error: %s\n", mpv_error_string(status));
+        exit(1);
+    }
+}
+
+static void *get_proc_address(void *ctx, const char *name)
+{
+    CFStringRef symbolName = CFStringCreateWithCString(kCFAllocatorDefault, name, kCFStringEncodingASCII);
+    void *addr = CFBundleGetFunctionPointerForName(CFBundleGetBundleWithIdentifier(CFSTR("com.apple.opengles")), symbolName);
+    CFRelease(symbolName);
+    return addr;
+}
+
+static void glupdate(void *ctx);
+
+@interface MpvClientOGLView : GLKView
+    @property mpv_opengl_cb_context *mpvGL;
+@end
+
+@implementation MpvClientOGLView {
+    GLint defaultFBO;
+}
+    
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+    self.context = [[EAGLContext alloc] initWithAPI:kEAGLRenderingAPIOpenGLES3];
+    [EAGLContext setCurrentContext:self.context];
+
+    // Configure renderbuffers created by the view
+    self.drawableColorFormat = GLKViewDrawableColorFormatRGBA8888;
+    self.drawableDepthFormat = GLKViewDrawableDepthFormatNone;
+    self.drawableStencilFormat = GLKViewDrawableStencilFormatNone;
+    
+    defaultFBO = -1;
+}
+    
+- (void)fillBlack
+{
+    glClearColor(0, 0, 0, 0);
+    glClear(GL_COLOR_BUFFER_BIT);
+}
+    
+- (void)drawRect
+{
+    if (defaultFBO == -1)
+    {
+        glGetIntegerv(GL_FRAMEBUFFER_BINDING, &defaultFBO);
+    }
+
+    if (self.mpvGL)
+    {
+        mpv_opengl_cb_draw(self.mpvGL,
+                           defaultFBO,
+                           self.bounds.size.width * self.contentScaleFactor,
+                           -self.bounds.size.height * self.contentScaleFactor);
```

Comment:
Nope, that doesn't work well 😕 

Using bounds width/height:
![image](https://user-images.githubusercontent.com/1083228/28243249-8869f264-69c3-11e7-83a6-b842f18f472d.png)

Using drawableWidth/Height:
![image](https://user-images.githubusercontent.com/1083228/28243258-b5e1e652-69c3-11e7-95c8-b9afa733b978.png)